Readme.md

```diff
@@ -100,20 +98,20 @@ It supports the following output formats:
 
 **Options**
 
-| Property          | Type   | Description                                                                             |
-|-------------------|--------|-----------------------------------------------------------------------------------------|
-| decimalPlaces | `number`｜`null` | Maximum number of decimal places to include in output. Default value to `2`. |
-| fixedDecimals | `boolean`｜`null` | Whether to always display the maximum number of decimal places. Default value to `false` |
-| thousandsSeparator | `string`｜`null` | Example of values: `' '`, `','` and `.`... Default value to `''`. |
-| unit | `string`｜`null` | The unit in which the result will be returned (B/KB/MB/GB/TB). Default value to `''` (which means auto detect). |
-| unitSeparator | `string`｜`null` | Separator to use between number and unit. Default value to `''`. |
-| mode         | `string`&124;`null` | Which format to output: `binary`, `metric`, `decimal`, `jedec`, `compatibility`. Default value is `metric` |
+| Property          | Type   | Description                                                                             | Default |
+|-------------------|--------|-----------------------------------------------------------------------------------------|---------|
+| decimalPlaces | `number`｜`null` | Maximum number of decimal places to include in output | 2 |
+| fixedDecimals | `boolean`｜`null` | Whether to always display the maximum number of decimal places, i.e. preserve trailing zeroes | `false` |
+| thousandsSeparator | `string`｜`null` | What to separate large numbers with, e.g. `','`, `'.'`, `' '`, ... | `''` |
+| unit | `string`｜`null` | The unit in which the result will be returned: `'B'`, `'kB'`, `'KiB'`, ... | `''` (autodecide) |
```

Comment:
I wouldn't use 'autodecide' thats a very "user oriented" word, "autodetect" is industry standard for parsing and such

Reply:
But this is formatting. There's nothing to "detect"

Reply:
Autoscale?

Reply:
"detect appropriate unit to format to" still fits the common industry usage.  But instead we can clarify what the option is "largest possible" perhaps would fit?

Reply:
> `''` (largest possible)

Doesn't really click... I'll just leave it on autodetect to cut the knot